uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/manage-leave-requests.component.js

```diff
@@ -307,19 +314,24 @@ define([
      */
     function loadLeaveRequests (type) {
```

Comment:
`type` should be something like `section`. We are using the wrong terminology here.